radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNetworkController.java

```diff
@@ -120,8 +120,14 @@ private RadixNetworkController(
 			.map(epic -> epic.epic(connectableReducedNodeActions, networkState))
 			.collect(Collectors.toSet());
 
-		// FIXME: Cleanup disposable
-		Observable.merge(updates).subscribe(this::dispatch);
+		// FIXME: Cleanup disposable and error handling
+		Observable.merge(updates).subscribe(
+			this::dispatch,
+			e -> {
+				e.printStackTrace();
```

Comment:
Do we have a dedicated logger (like `slf4j` or similar) that could/would/should handle the printing of this exception instead? (Benefit being that the logger can be configured "globally" to control output to some degree, while `printStackTrace()` doesn't honour any of those configurations).

Reply:
good point